mod/admin.php

```diff
@@ -282,14 +282,14 @@ function admin_page_federation(&$a) {
 	foreach ($platforms as $p) {
 		// get a total count for the platform, the name and version of the
 		// highest version and the protocol tpe
-		$c = q('SELECT COUNT(*) AS `total`, `platform`, `network`, `version` FROM `gserver`
+		$c = qu('SELECT COUNT(*) AS `total`, `platform`, `network`, `version` FROM `gserver`
 				WHERE `platform` LIKE "%s" AND `last_contact` > `last_failure` AND `version` != ""
```

Comment:
There are many queries in the code that don't use uppercase for keywords, don't use backticks, select "*" instead of the needed fields, ...

It surely will be rewritten some time in the future. But when I simply exchange a function call with another one in several files then I will not have a deep look at every single file.